core/quill.js

```diff
@@ -467,6 +469,27 @@ Quill.events = Emitter.events;
 Quill.sources = Emitter.sources;
 // eslint-disable-next-line no-undef
 Quill.version = typeof QUILL_VERSION === 'undefined' ? 'dev' : QUILL_VERSION;
+Quill.LIST_STYLE_KEY = 'mso-list-data';
+
+Quill.replaceStyleAttribute = (html) => {
+  const tagAttrsRegex = /(?:(<[a-z0-9]+\s*))([\s\S]*?)(>|\/>)/gi;
+
+  return html.replace(tagAttrsRegex, (allTagAttrs, tagStart, tagAttrs, tagEnd) => {
+    const contentWithoutStyle = tagAttrs.replace(/style(\s?)+=/gi, `${STYLE_ATTRIBUTE_KEY}=`);
```

Comment:
Isn't `/style(\s?)+=/gi` the same as `/style\s*=/gi`. The second option seems to be simpler. But it's up to u

Reply:
thanks. fixed